### ISSUE TITLE
example fails on Julia 1.8.1

### ISSUE THREAD

**Issue by touchft** [julia-vscode/ChromeProfileFormat.jl#21]
Try example on Julia 1.8.1

```julia
using Profile, ChromeProfileFormat

function foo()
    A = randn(1000, 1000)
    return inv(A)
end

foo()
Profile.clear()
@profile foo()
ChromeProfileFormat.save_cpuprofile("test.cpuprofile", from_c=true)
```

get error
```julia
ERROR: KeyError: key 0x0000000000000002 not found
Stacktrace:
 [1] getindex
   @ ./dict.jl:498 [inlined]
 [2] ChromeProfileFormat.CPUProfile(data::Nothing, period::Nothing; from_c::Bool)
   @ ChromeProfileFormat ~/.julia/packages/ChromeProfileFormat/UAHXj/src/core.jl:116
 [3] #save_cpuprofile#8
   @ ~/.julia/packages/ChromeProfileFormat/UAHXj/src/core.jl:195 [inlined]
 [4] top-level scope
   @ ~/tmp/test.jl:37
```

**Comment by fredrikekre** [comment]
Try the master branch, I think it was fixed by https://github.com/julia-vscode/ChromeProfileFormat.jl/pull/20.

**Comment by touchft** [comment]
> Try the master branch, I think it was fixed by #20.

Yes.  It was fixed!

**Comment by davidanthoff** [comment]
I'm tagging a new version now.
